app/models/operation.rb

```diff
@@ -3,7 +3,7 @@ class Operation < ApplicationRecord
 
   validates :job_class, presence: true
   validates :creator, presence: true
-  validates :description, presence: true
+  validates :details, presence: true
```

Comment:
Let's please remove _all_ these validations (not just the new one you added).

Validations only make sense if there is a form involved, and there is not in this case.